components/src/dynamo/sglang/main.py

```
@@ -238,20 +263,42 @@ async def init_prefill(runtime: DistributedRuntime, config: Config):
+            list_loras_endpoint.serve_endpoint(
+                handler.list_loras,
+                metrics_labels=metrics_labels,
+            ),
+            register_llm_with_readiness_gate(
+                engine,
+                generate_endpoint,
+                server_args,
+                dynamo_args,
+                input_type=ModelInput.Tokens,
+                output_type=ModelType.Prefill,
+                readiness_gate=ready_event,
```

Ruff (0.14.7)

Undefined name `ready_event`

(F821)